lib/directory-tree.js

```diff
@@ -12,9 +12,8 @@ function safeReadDirSync (path) {
 	try {
 		dirData = FS.readdirSync(path);
 	} catch(ex) {
-		if (ex.code == "EACCES")
-			//User does not have permissions, ignore directory
-			return null;
+		//User does not have permissions, ignore directory
+		if (ex.code == "EACCES" || ex.code == "EPERM") return null;
```

Comment:
Please keep the coding style as-is (`return null`) on the separate line.

Reply:
I changed it to the previous coding style